montie/session_03/mailroom.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+donors = ["Don Rickles","Mitch Hedberg","Dean Martin","Joan Rivers","Charles Rocket"]
+donation_amount = ["50:20:100","70","491:36","10:50:71","87:92"]
+
+
+def send_email(amount_donated,contributor):
+	total = 0
+	for num in amount_donated.split(':'):
+		total += int(num)	
+	print("Thank You {name} for your generous contributions.".format(name=contributor))
+	print("Your most recent donation of {} is very generous and we appreciate your overall contribution of {}.".format(amount_donated.split(':')[-1],total))
+	print("Sincerely, this org.")
+	
+
+def send_menu():
+		send_name = input("Type in a name: ")
+		if send_name == "list":
+			print("\n".join(donors))
+			send_menu()
+		if send_name in donors:
+			don_amount = input("How much did they donate? ")
+			daindex = donors.index(send_name)
+			if donation_amount[daindex] == "0":
+				donation_amount[daindex] = don_amount
+			else:
+				donation_amount[daindex] = donation_amount[daindex] + ":" + don_amount
+			total_donations = donation_amount[daindex]
+			send_email(total_donations,send_name)
+		if send_name not in donors:
+			donors.append(send_name)	
+			donation_amount.append("0")
+			print("Not in Donor's list; added.")
+			send_menu()			
```

Comment:
I believe the requirements state to ask for a donation as soon as they are added.  But the approach works either way.